xarray/plot/plot.py

```diff
@@ -567,6 +601,16 @@ def imshow(x, y, z, ax, **kwargs):
 
     Wraps :func:`matplotlib:matplotlib.pyplot.imshow`
 
+    While other plot methods require the DataArray to be strictly
+    two-dimensional, ``imshow`` also accepts a 3D array where some
+    dimension can be interpreted as RGB or RGBA color channels and
+    allows this dimension to be specified via the kwarg ``rgb=``.
+    In this case, ``robust=True`` will saturate the image in the
+    usual way, consistenly between all bands and facets.
+
+    This method will clip oversaturated pixels to the valid range,
```

Comment:
Is this actually strictly better? If so, why didn't it get changed during matplotlib's recent style overhaup? Was it simply overlooked or a more careful decision?

Reply:
IMO it is strictly better, because it avoids the ambiguity a colour could actually represent *any value at all* modulo the range.

I believe it was overlooked because relatively few people start with crudely-scaled RGB images, though I wasn't involved closely enough to know for sure (or fix it then!).  Unfortunately such "scaling" is pretty common to go from albedo to useful dynamic range in imagery, as I mentioned about `robust` above.